morituri/image/image.py

```diff
@@ -250,6 +250,7 @@ def stop(self):
                 self.setException(taskk.exception)
                 break
 
+            assert taskk.length is not None, "Track length was not found; look for earlier errors in debug log (set RIP_DEBUG=4)"
```

Comment:
Can this be broken up into two lines to help us pretend we're going to follow PEP8 at some point?